pkg/auth/authproviders/oidc/backend_impl.go

```diff
@@ -699,6 +713,18 @@ func mapCustomClaims(externalUserClaim *tokens.ExternalUserClaim, mappings map[s
 	return nil
 }
 
+func claimsAsString(claimExtractor claimExtractor) (string, error) {
+	claims := make(map[string]interface{})
+	if err := claimExtractor.Claims(&claims); err != nil {
+		return "", errors.Wrap(err, "failed to extract claims from IdP's token")
```

Comment:
```suggestion
		return "", errors.Wrap(err, "failed to extract all claims")
```
can be both the token and userinfo response